src/backend/joanie/settings.py

```diff
@@ -474,6 +475,9 @@ class Base(Configuration):
     CELERY_BROKER_URL = values.Value("redis://redis:6379/0")
     CELERY_BROKER_TRANSPORT_OPTIONS = values.DictValue({})
     CELERY_RESULT_BACKEND = values.Value("redis://redis:6379/0")
+    CELERY_IMPORTS = (
+        "joanie.edx_imports.tasks.universities",
+    )
```

Comment:
IMO if in the module `tasks.__init__` you import the tasks here there is no need to set this setting

Reply:
Nice idea :+1: